drivers/ieee802154/ieee802154_mcr20a.c

```diff
@@ -202,12 +203,13 @@ bool _mcr20a_write_burst(struct mcr20a_spi *spi, bool dreg, u16_t addr,
 {
 	bool retval;
 
-	k_sem_take(&spi->spi_sem, K_FOREVER);
-
 	if ((len + 2) > sizeof(spi->cmd_buf)) {
```

Comment:
I'm assuming my PR will be applied over this one to fix the condition test here. If not, I would suggest going ahead and fixing the condition test to be >=.

Reply:
@dleach02 can you please pick the 114d671 and rebase your PR #4575 on it, thx